src/main/java/project/Application.java

```diff
@@ -0,0 +1,22 @@
+package project;
+
+import org.springframework.boot.orm.jpa.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
+/**
+ * Created by slava23 on 10/11/2016.
+ */
+
+@SpringBootApplication
+@EnableJpaRepositories(basePackages = {"project/model"})
```

Comment:
@ComponentScan is part of @SpringBootApplication for default package only. If you need to scan custom packages - there is need to override it.